dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -527,60 +508,29 @@ public static void applyJsonFields(String value, Boolean omitJsonFields) {
 
             //env var omitJsonFields
             if (pair.trim().endsWith(":") && omitJsonFields) { //FIND FIELDS THAT NEED TO BE OMITTED
```

Comment:
I think you could avoid a ton of new code if you just look at pair.trim(), and if it ends with ":" recreate entry array with an extra array element containing OMIT_FIELDS_STRING